sql/dd_table_share.cc

```diff
@@ -1187,6 +1187,16 @@ static bool fill_index_from_dd(THD *thd, TABLE_SHARE *share,
       break;
   }
 
+  /* Check if this index is of clustering key type. Used by TokuDB */
+  const dd::Properties *index_options = &idx_obj->options();
```

Comment:
nit: why not a reference instead of pointer?

Reply:
can be written that way too. Updated the patch